curator-framework/src/test/java/org/apache/curator/framework/FrameworkTest.java

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
I don't see enough value in this class to maintain it. In order for it to be used in the recipes module we'd have to build a test jar and include and/or create another test module. Create new clients isn't always consistent in the tests and it's only 1 line of code anyway. The closeable stuff could be pushed into BaseClassForTests by having a List<Closeable> that can be added to and auto closed on @After.